NewHorizons/Utility/Files/ImageUtilities.cs

```diff
@@ -101,10 +101,6 @@ public static void ClearCache()
             _textureCache.Clear();
         }
 
-        /// <summary>
-        /// used specifically for projected slides.
-        /// also adds a border (to prevent weird visual bug) and makes the texture linear (otherwise the projected image is too bright).
```

Comment:
why remove comment about border, isnt there still border?

Reply:
i put that comment there because the method used to be called "invert image" so i wanted to specify that it was not a general invert. but now its named invert slide reel so i thought that wasnt needed. ill put it back

Reply:
yeah the border thing is just a rly weird solution so good to keep a comment explaining it